hack/jenkins/build.sh

```diff
@@ -33,6 +33,12 @@ export HOME=${WORKSPACE} # Nothing should want Jenkins $HOME
 export PATH=$PATH:/usr/local/go/bin
 export KUBE_SKIP_CONFIRMATIONS=y
 
+# Configure a temprary GOPATH.  The various go tools require this.
+mkdir -p .gopath/src/k8s.io
+ln -s $(pwd) .gopath/src/k8s.io/kubernetes
```

Comment:
I remember `go` having issues with symlinks, but I don't remember the details anymore. hopefully this works.